src/index.js

```diff
@@ -1,92 +1,172 @@
+  // userNameInput.addeventlistener('submit', function(e){console.log(e.target.value); e.target.value = '';})
+  // scheduleRequest.addeventlistener('submit', renderSchedule(Schedule))
+
+  window.onload = () => {
+    drawGraph()
+  }
+
+  let drawGraph = () => {
+    // deletes graph if already present
+    if(contributionGraph.hasChildNodes()) {
+      while(contributionGraph.firstChild) {
+        contributionGraph.firstChild.remove();
       }
     }
-    // APPEND WEEK TO GRAPH
-    contributionGraph.appendChild(createdWeek);
+    // draws graph in UI based off current graphState
+    for(let Week in GraphStateCtrl.graphState) {
+      let currentlyEvaluatedWeek = GraphStateCtrl.graphState[Week];
+      const createdWeek = document.createElement("div");
+      createdWeek.id = Week;
+      let day = 0;
+      currentlyEvaluatedWeek.forEach(dayValue => {
+        const createdDay = document.createElement("div");
+        createdDay.id = day;
+        createdDay.className = `commit-${dayValue}`;
+        createdWeek.appendChild(createdDay);
+        day++
+      });
+      contributionGraph.appendChild(createdWeek);
+    };
+  }
+
+  let graphClear = (event) => {
+    event.preventDefault()
+    GraphStateCtrl.initGraph(0);
+    drawGraph();
+  }
+
+  // update to include change in color over time held
+  let drawSwitch = true;
+  let eraseSwitch = false;
+  let fullSwitch = false;
+  let drawOptionSwitcher = (event) => {
+    let optionToTurnOn = event.target.id.replace('-option', 'Switch');
+    if(optionToTurnOn === "eraseSwitch") {
+      eraseSwitch = true;
+      fullSwitch = false;
+      drawSwitch = false;
+    } else if(optionToTurnOn === "fullSwitch") {
+      fullSwitch = true;
+      eraseSwitch = false;
+      drawSwitch = false;
+    } else if(optionToTurnOn === "drawSwitch") {
+      drawSwitch = true;
+      eraseSwitch = false;
+      fullSwitch = false;
+    }
+  }
+  // not very nice code due to repetition but I think it is clearer in proceeding code compared to an object holding these values
 
-    // CREATE WEEK EVENT LISTENER
-    document
-      .getElementById(`week-${week}`)
-      .addEventListener("click", weekClick);
+  let mouseDownUpDraw = () => {
+    drawHandler();
+    drawEraseFull(event);
   }
-};
-
-textForm.addEventListener("submit", function (e) {
-  console.log(textField.value);
-  e.target.children[0].children[1].value = "";
-  // input - function that takes letter and inputs it to the commitGraphState and the cubes
-  // input - function that creates the schedule output
-  e.preventDefault();
-});
-
-function weekClick(e) {
-  if (e.target.id.includes("day")) {
-    e.target.className = "commit-2";
-  } else {
+  let mouseOverDraw = (event) => {
+    drawEraseFull(event);
+  }
+
+  let draw = false;
+  let drawHandler = () => draw = !draw;
+
+  let drawEraseFull = (event) => {
+    let currentCommitValue = parseInt(event.target.className.charAt(7));
+    if(drawSwitch === true && draw === true) {
```

Comment:
Typically it's totally fine to assert boolean holding variables without `===`, so 
```
if(drawSwitch && draw ) {
```
Would be the equivalent.

Another unrelated note would be some feedback regarding the naming of the boolean holding variables.

To understand `drawSwitch`, my full train of thought is like, "ah so it's used as some kind of switching mechanism that is analogous to having a pencil touching the paper or not touching the paper. One will draw and one will not."

But this is slightly inefficient for the reader since I'm dwelling on the switch mechanism way too much. `isDrawing` is a lot more succinct. In fact with boolean variable naming conventions, `is[verb]` is a fantastic naming convention. It does a good job at abstracting away the details and focusing on what is happening rather than how it's happening. This allows the reader to at a glance understand a little quicker what is going on. But also having a convention for boolean variables makes naming variables in the future a little easier too.